apps/src/codebridge/hooks/useInitialSources.ts

```diff
@@ -72,9 +54,12 @@ export const useInitialSources = (defaultSources: ProjectSources) => {
       // If we have a serialized maze, we need to add it to the project sources so
       // it is accessible when running/sharing/remixing the code. We save it as a
       // system support file.
-      if (serializedMaze) {
+      if (levelProperties.serializedMaze) {
```

Comment:
nit/optional: destructure `levelProperties` above to avoid having to reference each time?

Reply:
Yeah I debated this, I'll add the destructure because it'll be a good pattern to follow to avoid even more changes in future prs.